app/layout.tsx

```diff
@@ -86,7 +88,7 @@ export default function RootLayout({
   }, []);
 
   return (
-    <html lang="en">
+    <html lang="en" id="html">
```

Comment:
What does `id="html"` do? 
It doesn't seem to have any styling attached and doesn't seem to make a difference if it's there or not?